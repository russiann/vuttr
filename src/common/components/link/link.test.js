import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Link from './link.component';

afterEach(cleanup);

it('renders', () => {
  const {asFragment} = render(<Link href="http://www.google.com">Google</Link>);
  expect(asFragment()).toMatchSnapshot();
});
