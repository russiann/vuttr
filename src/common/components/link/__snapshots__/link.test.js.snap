// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <a
    class="css-1n4a4ni-Link esh20tv0"
    href="http://www.google.com"
  >
    Google
  </a>
</DocumentFragment>
`;
